websites/A/AnimeGO/presence.ts

```diff
@@ -0,0 +1,142 @@
+const presence = new Presence({
+	clientId: "935597176426491924"
+});
+
+async function getStrings() {
+	return presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused"
+	});
+}
+
+let strings: Awaited<ReturnType<typeof getStrings>>,
+	video = {
+		duration: 0,
+		currentTime: 0,
+		paused: true
+	};
+
+presence.on(
+	"iFrameData",
+	(data: { duration: number; currentTime: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		details: "Где-то на сайте",
```

Comment:
why are you assigning details at the very beginning of the event just to replace them later?

Reply:
This is the default value if the `details` won't be set later.

Reply:
I know, I'm saying that approach is not very verbose

Reply:
Google translates this as "Somewhere on the site", but the correct translation in this case is "Browsing some page on the site". It's understandable for Russians.